DQM/SiPixelPhase1Summary/interface/SiPixelPhase1Summary.h

```diff
@@ -69,10 +70,20 @@
 
        std::map<std::string,std::string> summaryPlotName_;
 
+       //The dead and innefficient roc trend plot
+       std::map<std::string,MonitorElement*>  deadROCTrends_;
```

Comment:
Hi @leggat - please change these to be std::map<enum,MonitorElement*> to avoid all the string lookups below. (where you should define a set of enums to meet your needs) [best to do this also with the key to summaryPlotName_] Thanks